README.md

```diff
@@ -115,9 +115,12 @@ $ make test
 The majority of tests in the provider are `Acceptance Tests` - which provisions real resources in Azure. It's possible to run the entire acceptance test suite by running `make testacc` - however it's likely you'll want to run a subset, which you can do using a prefix, by running:
 
 ```sh
-make acctests SERVICE='resource' TESTARGS='-run=TestAccAzureRMResourceGroup' TESTTIMEOUT='60m'
+make acctests SERVICE='<resource>' TESTARGS='-run=<nameOfTheTest>' TESTTIMEOUT='60m'
```

Comment:
```suggestion
make acctests SERVICE='<service>' TESTARGS='-run=<nameOfTheTest>' TESTTIMEOUT='60m'
```
Maybe name this service, rather than a resource to avoid confusion with the actual individual resources?